### ISSUE TITLE
Recorder starting when a new connection is made to a camera

### ISSUE THREAD

**Issue by lolorc** [roflcoopter/viseron#518]
using dev,
no object detection, but using mog2
~~no issue if I disable mog2~~ => can't tell, the camera won't start with no motion and no object detection :-)

**Comment by roflcoopter** [comment]
Yeah this happens because motion detection works by comparing the previous frame to the current one. and on first start the previous frame is empty (or all black) which makes it trigger the recorder.

Will see if i can do something about that.

**Comment by lolorc** [comment]
skip comparison if no previous frame ? or if frame count ==1 ? 
I could probably test a provide a PR, but atm I'm struggling to tune motion detection (too many false detections, unable to detect my cat moving)

**Comment by roflcoopter** [comment]
Yeah thats how it would be done, i didnt want to include it since it would be unnecessary for performance reasons but it wont matter much at all

**Comment by roflcoopter** [comment]
This is fixed in the latest version
